### PR TITLE
Drop suggestion reject subs

### PR DESCRIPTION
**Pull request by phlax**
these dont contain any data that is not in the suggestion so this PR drops this sub type.

It also temporarily disables scoring/timeline from suggestion rejection actions

### CODE REVIEW

File: pootle/apps/pootle_statistics/migrations/0017_drop_reject_suggestion_subs.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-02-28 13:11
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db.models import F
```
Comment (unho) [inline pootle/apps/pootle_statistics/migrations/0017_drop_reject_suggestion_subs.py]: Unused import.

File: pootle/apps/pootle_statistics/models.py [hunk 2]
```diff
@@ -232,10 +231,10 @@ def get_submission_info(self):
                                                           check_name),
                     'checks_url': reverse('pootle-checks-descriptions'),
                 })
-
-        if (self.suggestion and
-            self.type in (SubmissionTypes.SUGG_ACCEPT,
-                          SubmissionTypes.SUGG_REJECT)):
+        is_suggestion = (
+            self.suggestion
+            and (self.type == SubmissionTypes.SUGG_ACCEPT))
```
Comment (unho) [inline pootle/apps/pootle_statistics/models.py]: Nit: don't use unnecessary parentheses.

File: pootle/apps/pootle_statistics/models.py [hunk 3]
```diff
@@ -563,8 +556,6 @@ def get_sugg_accepted():
             TranslationActionCodes.REVIEW_PENALTY: lambda: (-1) * reviewCost,
             TranslationActionCodes.SUGG_ACCEPTED: get_sugg_accepted,
             TranslationActionCodes.SUGG_REVIEWED_ACCEPTED: lambda: reviewCost,
-            TranslationActionCodes.SUGG_REJECTED: get_sugg_rejected,
-            TranslationActionCodes.SUGG_REVIEWED_REJECTED: lambda: analyzeCost,
```
Comment (unho) [inline pootle/apps/pootle_statistics/models.py]: Are these two fields of `TranslationActionCodes` still needed?

File: pootle/apps/pootle_statistics/models.py (hunk 2 above)
Comment (phlax) [inline pootle/apps/pootle_statistics/models.py]: in this case - i prefer a little extra bracket as i feet it makes the condition clearer

File: pootle/apps/pootle_statistics/models.py (hunk 3 above)
Comment (phlax) [inline pootle/apps/pootle_statistics/models.py]: im not sure we are going to use in the future - but the scorelog entries that contained them would be auto removed

**Pull request merged**